src/tools/Brush.js

```diff
@@ -18,14 +20,9 @@ const ToolView = observer(({ item }) => {
         item.manager.unselectAll();
 
         item.setSelected(!sel);
-
-        if (item.selected) {
-          item.updateCursor();
-        }
       }}
       onChange={val => {
         item.setStroke(val);
-        item.updateCursor();
```

Comment:
It just wasn't in the right place. In case when we need to update the cursor on each change in `selected` and `strokeWidth`, we have to write `updateCursor` in couple with calling `setSelected `and `setStroke`, every time. But I think I found the better way, than reactions, to fix this.